.github/workflows/build.yml

```diff
@@ -0,0 +1,72 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Unit Tests
+
+    strategy:
+      matrix:
+        node-version:
+          - 12.x
+          - 14.x
+        os:
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+    
+      - run: npx lerna bootstrap && npx lerna run build
+      - run: npm run build --if-present
+      - run: npm test
+
+  publish:
+    needs:
+      - build
+      - notify
```

Comment:
Looks really good @rkotangoor ! This `- notify` dependency will have to be removed since we deleted the `notify` part of the workflow.